src/scraper/items.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+
+# Define here the models for your scraped items
+#
+# See documentation in:
+# http://doc.scrapy.org/en/latest/topics/items.html
+
+import scrapy
+
+
+class ScraperItem(scrapy.Item):
+    # define the fields for your item here like:
+    # name = scrapy.Field()
+    pass
```

Comment:
Well if it's generated but must be added to the source, put it all in a totally different directory. Like `vendor/` or `generated/` or `third-party/`